pkg/querier/astmapper/parallel.go

```diff
@@ -0,0 +1,108 @@
+package astmapper
+
+import (
+	"fmt"
+
+	"github.com/go-kit/log/level"
+	"github.com/prometheus/prometheus/promql/parser"
+
+	util_log "github.com/cortexproject/cortex/pkg/util/log"
```

Comment:
should this still be pointing to cortex?

Reply:
yes, until [this PR](https://github.com/grafana/loki/pull/4917) gets merged.